test/tests/functional/pbs_cgroups_hook.py

```diff
@@ -500,6 +500,16 @@ def setUp(self):
     devices_job="$devices_base/sbp.service/jobid/$PBS_JOBID"
 else
     devices_job="$devices_base/sbp.slice/sbp-${jobnum}.*.slice"
+fi
+
+device_list=`cat $devices_job/devices.list`
+grep "195" $devices_job/devices.list
```

Comment:
Couple of questions.  First do we want to hard code this?  Then you don't assign this grep into anything.  It'll just be printed to stdout.  Is that what you want?

Reply:
195 is the major number for nvidia devices, so we can hardcode this. We just want to print it out to stdout, the test will search for the output.